src/ScopedValues.jl

```diff
@@ -148,11 +171,25 @@ function Base.show(io::IO, val::ScopedValue)
     print(io, ')')
 end
 
+function restore(snapshot::Snapshot)
+    scope = nothing
+    for (var, val) in snapshot.values
+        scope = Scope(scope, var, val)
+    end
+    return scope::Scope
+end
```

Comment:
Isn't this purely `Scope(nothing, snapshort)`?

Reply:
I like having a function with a name like `restore`, even if its implementation is changed to just the constructor you identified.

Reply:
But it doesn't restore anything? It just contrstucts a Scope?